src/main/java/org/jabref/logic/importer/fetcher/AdvancedSearchConfig.java

```diff
@@ -0,0 +1,125 @@
+package org.jabref.logic.importer.fetcher;
+
+import java.util.Objects;
+import java.util.Optional;
+
+import org.jabref.model.strings.StringUtil;
+
+public class AdvancedSearchConfig {
+    private final String defaultField;
+    private final String author;
+    private final String title;
+    private final Integer fromYear;
+    private final Integer toYear;
+    private final String journal;
+
+    private AdvancedSearchConfig(String defaultField, String author, String title, Integer fromYear, Integer toYear, String journal) {
+        this.defaultField = defaultField;
+        this.author = author;
+        this.title = title;
+        this.fromYear = fromYear;
+        this.toYear = toYear;
+        this.journal = journal;
+    }
+
+    public Optional<String> getDefaultField() {
```

Comment:
What's the purpose of this "default field" ?

Reply:
The default field is used if no fielded search parameter is used. In most of the API's this results in searching in multiple (all) fields.

Reply:
Just add JavaDoc 😅

Reply:
Do you think this should be documented?